frameworks/PHP/workerman/setup.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+fw_depends mysql
+sudo add-apt-repository -y ppa:ondrej/php
+sudo apt-get update
+sudo apt-get -y --force-yes install php-cli php-mysql
```

Comment:
you should be able to remove lines 4-6 after using `fw_depends`

Reply:
it is not possible. "fw_depends php" does not have PCNTL capability.

Reply:
hmm.. will have to look into this. is it just simply a matter of installing it but leaving it out of `php.ini` for those other frameworks? We can also consider making a separate setup script for php7-pcntl

Reply:
I'll merge this in as is, just do me a favor and remove the apt repository in this script after you're done installing it.
`sudo add-apt-repository -y --remove ppa:ondrej/php`
and then we'll work on a better way to get this in.

Reply:
The PPA removal is added.

The PCNTL installation is just adding a parameter to
https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/toolset/setup/linux/languages/php/php7.sh#L23

```--enable-pcntl```
http://php.net/manual/en/pcntl.installation.php